Modules/Brand/Http/Services/BrandService.php

```diff
@@ -4,22 +4,22 @@
 
 use Illuminate\Support\Str;
 use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\File;
 use Illuminate\Support\Facades\Hash;
 use Illuminate\Support\Facades\Validator;
 use Illuminate\Validation\Rules\Password;
```

Comment:
do not need this anymore, delete

Reply:
comment was related to use Illuminate\Validation\Rules\Password; not use Illuminate\Support\Facades\File;
bring use Illuminate\Support\Facades\File; back and delete
use Illuminate\Validation\Rules\Password;

Reply:
Fixed